tests/drum/test_fit.py

```diff
@@ -175,6 +179,8 @@ def test_fit(
             target_type = BINARY
         elif problem == MULTICLASS_NUM_LABELS:
             target_type = MULTICLASS
+        elif framework == SKLEARN_TRANSFORM:
+            target_type = TRANSFORM
         else:
             target_type = problem
 
```

Comment:
in a separate PR, add a transform server test.

Reply:
In the server mode:
- do we use `predict` endpoint or we want to add `transform`?
- when transformer is used as a step in blueprint - how steps communicate to each other?

Reply:
the plan will probably be to stand up a transform endpoint (I'm looking into that right now as a follow-up). I need to do a little more reading up on customizable blueprints, but my understanding is that there's a convention of a schema for all blueprint components, so a given step has expected input and output type that are validated. In terms of the actual mechanics of how they communicate- let me chat w/ some folks and get back to you

Reply:
...so it sounds like once the transformer is being used as a step in a blueprint, it's basically going to function like any other DR task in terms of how it communicates with up/downstream tasks (at least for now).

Reply:
and what does it mean: `function like any other DR task ` ? how does it communicate?